packages/runtimes/hmr/src/loaders/hmr-runtime.js

```diff
@@ -170,6 +170,39 @@ function getParents(bundle, id) {
   return parents;
 }
 
+function updateLink(link) {
+  var newLink = link.cloneNode();
+  newLink.onload = function() {
+    if (link.parentNode !== null) {
+      link.parentNode.removeChild(link);
+    }
+  };
+  newLink.setAttribute(
+    'href',
+    link.getAttribute('href').split('?')[0] + '?' + Date.now(),
+  );
+  link.parentNode.insertBefore(newLink, link.nextSibling);
+}
+
+var cssTimeout = null;
+function reloadCSS() {
+  if (cssTimeout) {
+    return;
+  }
+
+  cssTimeout = setTimeout(function() {
+    var links = document.querySelectorAll('link[rel="stylesheet"]');
+    for (var i = 0; i < links.length; i++) {
+      var absolute = /^https?:\/\//i.test(links[i].getAttribute('href'));
+      if (!absolute) {
+        updateLink(links[i]);
+      }
+    }
+
+    cssTimeout = null;
+  }, 50);
```

Comment:
what's the rationale for `50`? Can this be in a comment?

Reply:
This is just copied from Parcel 1: https://github.com/parcel-bundler/parcel/commit/2cc14c8da8a8b8b699e44f4f5fa87cc731e0ed68#diff-4471b25fd54ef2af9a791fe450751033
So we should ask @devongovett 😄